corehq/elastic.py

```diff
@@ -151,7 +151,15 @@ def run_query(index_name, q, debug_host=None):
     else:
         es_instance = get_es_new()
 
-    es_meta = ES_META[index_name]
+    try:
+        es_meta = ES_META[index_name]
+    except KeyError:
+        from corehq.apps.userreports.util import is_ucr_table
+        # todo: figure out if we really need types
+        if is_ucr_table(index_name):
+            es_meta = EsMeta(index_name, 'indicator')
```

Comment:
Didn't move this to helper as EsMeta is specifc to this file and I'm not convinced this is needed as I think an index will only have one datatype at any time.

Will probably rip out speicfying types in a later PR, but want to convince myself a little more